### ISSUE TITLE
SPSearchAuthoritativePage - Demoted Site Already Exists

### ISSUE THREAD

**Issue by ruandersMSFT** [dsccommunity/SharePointDsc#1167]
#### Details of the scenario you tried and the problem that is occurring

Running SPSearchAuthoritativePage with Action = 'Authoratative' works fine.  Running with Action = 'Demoted' works fine the first time, but subsequent runs fail with message that the Demoted Site already exists.

#### Verbose logs showing the problem

![image](https://user-images.githubusercontent.com/16927960/74591303-1445c800-4fd4-11ea-9f4a-41ec2b5d7d94.png)

#### Suggested solution to the issue

Likely a state comparison bug in current vs target values.  Recommend review of value/state comparison.

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)

```powershell
SPSearchAuthoritativePage DemotedPage
{
	ServiceAppName = 'Search Service Application'
	Path = 'https://www.contoso2.net/'
	Level = 0.66
	Action = 'Demoted'
	Ensure = 'Present'
	DependsOn = "[SPSearchServiceApp]MySearchService"
}
```

#### The operating system the target node is running

OsName               : Microsoft Windows Server 2019 Datacenter
OsOperatingSystemSKU : DatacenterServerEdition
OsArchitecture       : 64-bit
WindowsVersion       : 1809
WindowsBuildLabEx    : 17763.1.amd64fre.rs5_release.180914-1434
OsLanguage           : en-US
OsMuiLanguages       : {en-US}

#### Version of SharePoint that is used (e.g. SharePoint 2016)

SharePoint 2019

#### Version and build of PowerShell the target node is running

Name                           Value                                                                                                                                                                                                                             
----                           -----                                                                                                                                                                                                                             
PSVersion                      5.1.17763.1007                                                                                                                                                                                                                    
PSEdition                      Desktop                                                                                                                                                                                                                           
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                                                           
BuildVersion                   10.0.17763.1007                                                                                                                                                                                                                   
CLRVersion                     4.0.30319.42000                                                                                                                                                                                                                   
WSManStackVersion              3.0                                                                                                                                                                                                                               
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                                               
SerializationVersion           1.1.0.1  

#### Version of the DSC module that was used

3.7

**Comment by ruandersMSFT** [comment]
Bug Identified in SPSearchAuthoritativePage, call to Get-TargetResource for a Demoted Action still contains a PowerShell pipe which is also passed as a Parameter.

![image](https://user-images.githubusercontent.com/16927960/77971049-f8a82f80-72aa-11ea-9e2e-ec08f7ed3391.png)

**Comment by ykuijs** [comment]
Fix will be included in the v4.0 release.
